### PR TITLE
Updated FriSon

### PR DESCRIPTION
**Pull request by MarkJaroski**
This one appears to have stopped working, so I simplified it.

There's a new field which I would like to propose for inclusion: image_url. This will save me the trouble of trying to guess this stuff from either the og: schema or the biggest image.

### CODE REVIEW

**Comment by jakopako** [comment]
Thanks for updating! There are some events that don't seem to have a genre which causes those events to be skipped. Is this intended? Also the time in the date is always 12:00

Adding the image url is perfect. I'll have to update the api to accept this field. Until then it will be ignored.

**Comment by MarkJaroski** [comment]
I'll look at the time again, and the genre. Sorry I missed those. And thanks for considering the image_url !

**Comment by MarkJaroski** [comment]
My bad. I was so happy to see that they used semantic markup for datetime that I overlooked the fact that all of their times are set to noon UTC. :(

**Comment by MarkJaroski** [comment]
It works correctly again.

File: concert-config.yml [hunk 1]
```diff
@@ -2011,15 +2011,15 @@ scrapers:
       - name: title
         type: text
         location:
-          selector: div.row_events > a.event.w-inline-block > div.event-lead > div.event-title-alt > h3.event-title-text
```
Comment (jakopako) [inline concert-config.yml:2014]: @MarkJaroski why did you update this line?

**Comment by jakopako** [comment]
Added imageUrl field to the api so once the scraper runs with this branch's config, Fri-Son events should contain an imageUrl field. Also added an offset field to be able to infer the correct local time.

File: concert-config.yml (hunk 1 above)
Comment (MarkJaroski) [reply concert-config.yml:2014]: Well, it looks like an editor accident. That's very sloppy work on my part. I should have caught that.

Comment (jakopako) [reply concert-config.yml:2014]: No worries :) i was just a little confused :D

**Pull request merged**